features/upgrade/workloads/descheduler-upgrade.feature

```diff
@@ -80,3 +80,42 @@ Feature: Descheduler major upgrade should work fine
       | n            | openshift-kube-descheduler-operator |
     Then the step should succeed
     And the output should match "clusterkubedescheduleroperator.*<%= cb.master_version %>.*Succeeded"
+
+  # @author knarra@redhat.com
+  # @case_id OCP-72824
+  @admin
+  @upgrade-check
+  @users=upuser1,upuser2
+  @destructive
+  @4.16 @4.15
+  @vsphere-ipi @openstack-ipi @nutanix-ipi @ibmcloud-ipi @gcp-ipi @baremetal-ipi @azure-ipi @aws-ipi @alicloud-ipi
+  @vsphere-upi @openstack-upi @nutanix-upi @ibmcloud-upi @gcp-upi @baremetal-upi @azure-upi @aws-upi @alicloud-upi
+  @upgrade
+  @network-ovnkubernetes @network-openshiftsdn
+  @proxy @noproxy @disconnected @connected
+  @heterogeneous @arm64 @amd64
+  @hypershift-hosted
+  @critical
+  Scenario: upgrade - upgrade descheduler from 4.x to 4.y
+    Given I switch to cluster admin pseudo user
+    And I use the "openshift-kube-descheduler-operator" project
+    Given I store master major version in the clipboard
+    Given a pod becomes ready with labels:
+      | app=descheduler |
+    Given I make sure the descheduler operator gets updated successfully if needed
+    And I use the "openshift-kube-descheduler-operator" project
+    Given I wait up to 180 seconds for the steps to pass:
+    """
+    And status becomes :running of exactly 1 pods labeled:
+      | name=descheduler-operator |
+    """
+    Given I wait up to 180 seconds for the steps to pass:
+    """
+    And status becomes :running of exactly 1 pods labeled:
+      | app=descheduler |
+    """
+    When I run the :get client command with:
+      | resource     | csv                                 |
+      | n            | openshift-kube-descheduler-operator |
+    Then the step should succeed
+    And the output should match "clusterkubedescheduleroperator.*v5.0.*Succeeded"
```

Comment:
Seems only the last output diffs for above two scenarios. Would you mind to try
```
Given evaluation of `env.version_ge("4.15", user: user) ? "xxx" : "yyy"` is stored in the :zzz clipboard
blabla-other-commands
And the output should match "<%= cb.zzz %> "
```
We can avoid to duplicate the test case and scripts in this way.

Reply:
sure, thanks !! I will update the script.. This is much better